$MirOS: ports/plan9/kencc/patches/patch-utils_va_a_h,v 1.1 2006/03/31 03:50:25 tg Exp $
--- utils/va/a.h.orig	Thu Mar 30 23:54:43 2006
+++ utils/va/a.h	Thu Mar 30 23:55:25 2006
@@ -118,7 +118,7 @@ EXTERN	Biobuf	obuf;
 
 void*	alloc(long);
 void*	allocn(void*, long, long);
-void	errorexit(void);
+void	errorexit(void) __attribute__((__noreturn__));
 void	pushio(void);
 void	newio(void);
 void	newfile(char*, int);
